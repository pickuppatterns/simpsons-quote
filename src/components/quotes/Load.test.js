import React from 'react';
import renderer from 'react-test-renderer';
import Load from './Load';
import { MemoryRouter } from 'react-router-dom';

describe('Load', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Load />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

