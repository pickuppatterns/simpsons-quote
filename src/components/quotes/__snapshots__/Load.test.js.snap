// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load matches a snapshot 1`] = `
<button>
  Next Quote
</button>
`;
